tests/virt_autotest/esxi_open_vm_tools.pm

```diff
@@ -144,7 +264,9 @@ sub login_vm_console {
     reset_consoles;
     console('svirt')->start_serial_grab;
     select_console('sut');
-    assert_screen('linux-login', 90);
+    assert_screen('grub2', 200);
+    wait_screen_change { send_key 'ret' };
```

Comment:
Hi Nan, would you please share the root cause for this? Why is it necessary now to interact with grub booting process?

Reply:
Because the grub timeout is disabled during installation, it requires to boot into OS while VM rebooting for checking vmtools service status.